### PR TITLE
Add http exporter

### PR DESCRIPTION
**Pull request by alexander-gr**

### CODE REVIEW

File: http_exporter.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    host: str = LKL_HOST,
+):
+    ''' Sends post query to the node via http.'''
+    resp = r.post(
+        url=host + method,
+        headers={'Content-Type': 'application/json'},
+        data=json.dumps(data)
+    )
+
+    if resp.status_code != 200:
```
Comment (ivanklimuk) [inline http_exporter.py]: I'd suggest to use
```assert respect.status_code == 200, error_message```

File: http_exporter.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        raise Exception('Invalid status code:', resp.status_code)
+    return json.loads(resp.text)
```
Comment (ivanklimuk) [inline http_exporter.py]: What's the difference between ```return resp.json()```

File: http_exporter.py [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    wait_block_policy: str = 'wait'
+):
+    ''' Gets block by number. If the the block is not mined yet and `wait_block_policy` == 'wait',
+        wait for new block `sleep_time` seconds and repeats itself.
+        If `wait_block_policy` != 'wait', stops the exporter.
+    '''
+    block_data = send_query('/get_block', {'number': b_number})
+
+    if block_data['status'] == 'error':
+        if block_data['result'] == f'Block was not found. number: {b_number}':
```
Comment (ivanklimuk) [inline http_exporter.py]: Maybe also change to `assert`?

File: http_exporter.py [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+            'fee': int(trx['fee']),
+            'dt': trx['timestamp'],
+            'depth': block_raw['depth']
+        })
+
+    return trx_data
+
+
+def find_trx_with_ts(hashes: list, ts: int):
+    ''' Iterates over all address's transactions and finds thansaction with timestamp corresponding to
```
Comment (ivanklimuk) [inline http_exporter.py]: `address's` -> `addresses'`
`thansaction` -> `transaction`

File: http_exporter.py [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+        current block's timestamp. In order to minimize amount of quiries to LikeLib node saves transaction
```
Comment (ivanklimuk) [inline http_exporter.py]: `queries` -> `queries`

**Review by ivanklimuk — APPROVED** [review]
LGTM, let's run it!

File: http_exporter.py (hunk 1 above)
Comment (MitriSan) [reply http_exporter.py]: Or even `assert resp.ok`

**Pull request merged**